extra/modules/pb-richmedia-filter/src/main/java/org/prebid/server/hooks/modules/pb/richmedia/filter/v1/PbRichmediaFilterBidResponsesFilterHook.java

```diff
@@ -0,0 +1,119 @@
+package org.prebid.server.hooks.modules.pb.richmedia.filter.v1;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.vertx.core.Future;
+import org.apache.commons.collections4.CollectionUtils;
+import org.prebid.server.auction.model.BidderResponse;
+import org.prebid.server.hooks.execution.v1.bidder.AllProcessedBidResponsesPayloadImpl;
+import org.prebid.server.hooks.modules.pb.richmedia.filter.core.BidResponsesMraidFilter;
+import org.prebid.server.hooks.modules.pb.richmedia.filter.model.AnalyticsResult;
+import org.prebid.server.hooks.modules.pb.richmedia.filter.model.MraidFilterResult;
+import org.prebid.server.hooks.modules.pb.richmedia.filter.v1.model.InvocationResultImpl;
+import org.prebid.server.hooks.modules.pb.richmedia.filter.v1.model.analytics.ActivityImpl;
+import org.prebid.server.hooks.modules.pb.richmedia.filter.v1.model.analytics.AppliedToImpl;
+import org.prebid.server.hooks.modules.pb.richmedia.filter.v1.model.analytics.ResultImpl;
+import org.prebid.server.hooks.modules.pb.richmedia.filter.v1.model.analytics.TagsImpl;
+import org.prebid.server.hooks.v1.InvocationAction;
+import org.prebid.server.hooks.v1.InvocationResult;
+import org.prebid.server.hooks.v1.InvocationStatus;
+import org.prebid.server.hooks.v1.analytics.Result;
+import org.prebid.server.hooks.v1.analytics.Tags;
+import org.prebid.server.hooks.v1.auction.AuctionInvocationContext;
+import org.prebid.server.hooks.v1.bidder.AllProcessedBidResponsesHook;
+import org.prebid.server.hooks.v1.bidder.AllProcessedBidResponsesPayload;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class PbRichmediaFilterBidResponsesFilterHook implements AllProcessedBidResponsesHook {
+
+    private static final String CODE = "pb-richmedia-filter-bid-responses-filter-hook";
+    private static final String ACTIVITY = "reject-richmedia";
+    private static final String SUCCESS_STATUS = "success";
+
+    private final ObjectMapper mapper;
+    private final BidResponsesMraidFilter mraidFilter;
+    private final boolean filterMraid;
+
+    public PbRichmediaFilterBidResponsesFilterHook(ObjectMapper mapper,
+                                                   BidResponsesMraidFilter mraidFilter,
+                                                   boolean filterMraid) {
+        this.mapper = Objects.requireNonNull(mapper);
+        this.mraidFilter = Objects.requireNonNull(mraidFilter);
+        this.filterMraid = filterMraid;
```

Comment:
Same here, `Objects.reqireNonNull`.

Reply:
it's a boolean, no chances to have null value